instagram/src/Components/PostContainer/PostPage.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PostContainer from './PostContainer';
+import SearchBar from '../SearchBar/SearchBar';
+import dummyData from '../../dummy-data';
+import igLogo from '../Authentication/ig_logo.svg';
+import './Post.css';
+import {SearchB} from '../../StyledComponents';
+import {InstaBar} from '../../StyledComponents'
+
+class PostPage extends React.Component{
+    constructor(){
+    super();
+        this.state={
+            data:[],
+            filteredUsers:[],
+            searchInput:''
+        }
+    }
+
+    componentDidMount(){
+        this.setState({
+          data: dummyData
+      })
+    }
+
+    changeHandler = e => {
+        this.setState({
+            searchInput: e.target.value
```

Comment:
I like that you're doing the changeHandler in a less dynamic way here, if only to also be aware that this is a common pattern.